packages/material-ui/src/Rating/Rating.test.js

```diff
@@ -74,6 +74,18 @@ describe('<Rating />', () => {
     expect(handleChange.args[0][1]).to.deep.equal(null);
   });
 
+  it('should allow to clear the rating by keyboard', () => {
+    const { container } = render(<Rating name="rating-test" value={1} />);
+
+    act(() => {
+      container.querySelector('#rating-test-1').focus();
+      fireEvent.keyDown(document.activeElement || document.body, { code: '37' });
```

Comment:
Does this work with `key` as well? Would be better to use `key` for readability.

Reply:
sure. updated it